src/synced_services/redmine/redmine_synced_service.ts

```diff
@@ -188,6 +189,21 @@ export class RedmineSyncedService implements SyncedService {
     return projects;
   }
 
+  private async _getProjectIds() {
+    const projects = await this._getAllProjects();
+    const projectIds: (string | number)[] = [];
+
+    if (Array.isArray(projects)) {
+
+      forEach(projects, function (project) {
+        projectIds.push(project.id)
+      })
+    } else {
+      return [];
+    }
```

Comment:
od řádku 194 až do sem lze nahradit za:
```js
return projects ? projects.reduce(project => project.id, []] : [];
```

Reply:
Ternar je bohuzel potreba protoze ta metoda ze ktery to taha projekty muze vratit i false. 

Reduce jsem snad spravne upravil aby si nestezoval na typy.

Reply:
To jo, já to myslel jako že místo ternáru dej klidně klasický `if` :)

Reply:
Současný kód za mě asi cajk, není to ideální, to by bylo, kdyby ta funkce vždy vracela array, jen případně prázdný, ale to by bylo na větší refaktor